notebooks/08A_composing_parallel_sorting_data.ipynb

```diff
@@ -321,59 +340,63 @@
     "            lambda tn: tn == n\n",
     "        ).all()\n",
     "    )\n",
-    ") for n in nodes}"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "# 3B. Compute Features from Performance Data using Queried Thickets"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 8,
-   "metadata": {},
-   "outputs": [],
-   "source": [
+    ") for n in nodes}\n",
+    "\n",
+    "# Re-compose quieried Thickets\n",
     "tk = th.Thicket.concat_thickets(\n",
     "    thickets=list(ntk_dict.values()),\n",
+    "    fill_perfdata=True,\n",
     ")\n",
-    "# Get mapping of node objects\n",
-    "node_objects = {n.frame[\"name\"]: n for n in [n for n in tk.dataframe.index.get_level_values(\"node\").unique()]}"
+    "# Drop duplicate profiles in the metadata from concat_thickets\n",
+    "unhashable_cols = [\"libraries\", \"cmdline\"] # Can't pass these cols in the check or error will be thrown. Won't change the outcome of the check\n",
+    "tk.metadata = tk.metadata.drop_duplicates(subset=[col for col in tk.metadata.columns if col not in unhashable_cols])"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 9,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# Drop duplicate profiles from concat_thickets\n",
-    "# Can't pass these cols in the check or error will be thrown. Won't change the outcome of the check\n",
-    "unhashable_cols = [\"libraries\", \"cmdline\"]\n",
-    "tk.metadata = tk.metadata.drop_duplicates(subset=[col for col in tk.metadata.columns if col not in unhashable_cols])"
+    "### 4B. Remove Profiles not Containing All Nodes\n",
+    "\n",
+    "Because our features will use all of the nodes, we remove profiles that do not have data for all nodes using `Thicket.filter_profile`."
```

Comment:
50e25978 I reworded this cell because it isn't necessary to think about nodes here but really we are just removing profiles with missing data.

There is still the issue of whether or not we want to explain the generalized calltree in this tutorial or not.